ts/src/test/static/request/kucoinfutures.json

```diff
@@ -182,7 +182,7 @@
             {
                 "description": "Swap private trades",
                 "method": "fetchMyTrades",
-                "url": "https://api-futures.kucoin.com/api/v1/fills?symbol=LTCUSDTM&startAt=1699457638000",
+                "url": "https://api-futures.kucoin.com/api/v1/fills?symbol=LTCUSDTM&startAt=1699457638000&pageSize=5",
```

Comment:
hum what is this?

Reply:
it was not implemented in fetchTrades, and i've added support for `limit` param

Reply:
@ttodua but why 5? that seems too small

Reply:
the test was using `5` as limit , it's just random static test

Reply:
i mean, if you look at 5 lines below, you'll see 'input' params

Reply:
![image](https://github.com/user-attachments/assets/604a0134-3fc0-4fbf-9dde-84c7a033f7b2)
 oh right, sorry my bad, I was assuming this was the "default" limit we were sending